sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1250,10 +1250,7 @@ object GpuOverrides {
       ExprChecks.unaryProjectNotLambdaInputMatchesOutput(TypeSig.STRING, TypeSig.STRING),
       (a, conf, p, r) => new UnaryExprMeta[InitCap](a, conf, p, r) {
         override def convertToGpu(child: Expression): GpuExpression = GpuInitCap(child)
-      }).incompat(CASE_MODIFICATION_INCOMPAT + " Spark also only sees the space character as " +
-      "a word deliminator, but this will capitalize any character after a non-alphabetic " + 
-      "character.  The behavior will be aligned to match Spark in the future per " +
-      "https://github.com/NVIDIA/spark-rapids/issues/2786."),
+      }).incompat(CASE_MODIFICATION_INCOMPAT),
```

Comment:
Is this still true?  `CASE_MODIFICATION_INCOMPAT` refers to https://github.com/rapidsai/cudf/issues/3132 which is a closed issue, and looking at the code it appears characters that change width are handled properly.  There may be some issues with corner-case Unicode characters based on which Unicode standard libcudf is using vs. the JVM is using, but I suspect the width-changing issue is no longer the problem.

Reply:
I spent some time to catch up with the story. 
Yes, There may be some issues caused by the different Unicode standards used by licudf and JVM respectively. Maybe we need another incompat message for it.
But here just removed this incompat message.